src/tool/cppwinrt/strings/base_meta.h

```diff
@@ -136,7 +136,11 @@ namespace winrt::impl
     template <typename T>
     struct guid_storage
     {
+#ifdef __clang__
+        inline static const guid value{ __uuidof(T) };
```

Comment:
While Clang generally honors MS compatibility, it doesn't extend to __uuidof. VC treats it as a constexpr expression but Clang does not. Fortunately, this only impacts classic COM interfaces and not WinRT types that do not depend on __uuidof.

Reply:
This seems worthy of a compatibility note in the code.

Reply:
Will add a comment. :thumbsup: